Dockerfile.windows

```diff
@@ -1,17 +1,24 @@
-FROM microsoft/dotnet:1.1-sdk-msbuild
+# escape=`
+FROM microsoft/dotnet:2.0-sdk-nanoserver
```

Comment:
Actually, for the dev branch it would be best to use `microsoft/aspnetcore-build-nightly:2`. This most closely represent the SDK our build scripts would have used to build the repo.